localisation:
    update_frequency: 50.0
    model_name: ngeeann_av

path_planner:
    update_frequency: 10.0
    frame_id: base_link

path_tracker:
    update_frequency: 50.0
    target_velocity: 5.0
    control_gain: 1.0
    softening_gain: 1.0
    yawrate_gain: 0.5
    steering_limits: 0.95
    centreofgravity_to_frontaxle: 1.483